src/Fixer/LanguageConstruct/FunctionToConstantFixer.php

```diff
@@ -167,7 +171,9 @@ private function fixFunctionCallToConstant(Tokens $tokens, int $index, int $brac
             $tokens->clearTokenAndMergeSurroundingWhitespace($i);
         }
 
-        if ($replacements[0]->isGivenKind([T_CLASS_C, T_STATIC])) {
+        if ($replacements[0]->isGivenKind([T_CLASS_C, T_STATIC])
+            || ($replacements[0]->isGivenKind(T_STRING) && 'self' === $replacements[0]->getContent())
+        ) {
```

Comment:
According to [PER Coding Style](https://www.php-fig.org/per/coding-style/#51-if-elseif-else), the conditions in the `if` must all be on their own lines when multiline (similar to e.g. function calls). I thought we had a rule for that.

Reply:
Done and fixer will be really welcomed as the "same startling line style" is quite common not only in general, but also in this repo.